src/Symfony/Component/Console/Command/Command.php

```diff
@@ -313,6 +313,10 @@ public function run(InputInterface $input, OutputInterface $output): int
 
         if ($input->isInteractive()) {
             $this->interact($input, $output);
+
+            if ($this->code?->isInteractive()) {
+                $this->code?->interact($input, $output);
```

Comment:
If `$this->code?->isInteractive()` is true, `$this->code` will necessarily exist, isn't it?

```suggestion
                $this->code->interact($input, $output);
```